tools/fuchsia/flutter/flutter_build_config.gni

```diff
@@ -0,0 +1,56 @@
+# Copyright 2020 The Fuchsia Authors. All rights reserved.
```

Comment:
I like this classification mechanism, makes it clearer than the obscure rules we have now. We do have more modes than what is enumerated below though, see: go/flutter-fuchsia-packaging.